code/modules/integrated_electronics/_defines.dm

```diff
@@ -56,7 +56,7 @@
 	var/next_use = 0                //Uses world.time
 	var/complexity = 1              //This acts as a limitation on building machines, more resource-intensive components cost more 'space'.
 	var/size                        //This acts as a limitation on building machines, bigger components cost more 'space'. -1 for size 0
-	var/cooldown_per_use = 1 SECOND // Circuits are limited in how many times they can be work()'d by this variable.
+	var/cooldown_per_use = 0.5 SECONDS // Circuits are limited in how many times they can be work()'d by this variable.
```

Comment:
undocumented change

Reply:
Yeah, this was just for testing. Didn't even change anything.
```suggestion
	var/cooldown_per_use = 1 SECOND // Circuits are limited in how many times they can be work()'d by this variable.
```